Library/Formula/openssl.rb

```diff
@@ -17,6 +18,9 @@ class Openssl < Formula
 
   depends_on "makedepend" => :build
 
+  keg_only :provided_by_osx,
```

Comment:
At the least this should be `keg_only :provided_pre_el_capitan` (the definition of which may need to be added) so this isn't keg-only there.